pkg/sql/logictest/testdata/logic_test/generators

```diff
@@ -234,3 +234,10 @@ SELECT * from generate_series(1, (select * from generate_series(1, 0)))
 query T
 SELECT unnest((select current_schemas((select isnan((select round(3.4, (select generate_series(1, 0)))))))));
 ----
+
+# Regression for #18021.
+query I
+SELECT GENERATE_SERIES(9223372036854775807::int, -9223372036854775807::int, -9223372036854775807::int)
+----
+9223372036854775807
+0
```

Comment:
I'm not sure that's correct. the right bound for `generate_series` is inclusive.
If we can't reach that value at all, then the call should produce an error instead.

Reply:
Good catch. Updated to fix this.